packages/ember-routing-htmlbars/lib/keywords/link-to.js

```diff
@@ -139,7 +139,35 @@ import merge from 'ember-metal/merge';
 
   To override this option for your entire application, see
   "Overriding Application-wide Defaults".
-
+  
+  ### Keeping a link active for other routes
+  
+  If you need a link to be 'active' even when it doesn't match
+  the current route, you can use the the `current-when`
+  argument.
+  
+  ```handlebars
+  {{#link-to 'photoGallery' current-when='photos'}}
+    Photo Gallery
+  {{/link-to}}
+  ```
+  
+  This may be helpful for keeping links active for:
+  * non-nested routes that are logically related
```

Comment:
Variable names that contain a dash need to be in quotes. Is that not the case here?